### PR TITLE
Fix semantics presubmit job.

### PR DESCRIPTION
**Pull request by droslean**

### CODE REVIEW

**Comment by bbguimaraes** [comment]
/lgtm

@stevekuznetsov: that will cost you a beer next week.

**Comment by droslean** [comment]
2 beers

File: hack/validate-prow-job-semantics.py [hunk 1]
```diff
@@ -60,7 +60,7 @@ def validate_file_structure(path):
             print("[ERROR] {}: file contains more than one type of job".format(path))
             return False
         if next(iter(data.keys())) == 'periodics':
-            return
```
Comment (petr-muller) [inline hack/validate-prow-job-semantics.py]: Ouch

File: projects/origin-release/python-validation/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,20 @@
+FROM centos
```
[the remark this answers is not in the document]
Comment (droslean) [reply projects/origin-release/python-validation/Dockerfile]: I will change with a lighter image.

**Pull request merged**